components/authz-service/storage/v2/postgres/postgres_test.go

```diff
@@ -6633,8 +6623,8 @@ func TestPurgeSubjectFromPolicies(t *testing.T) {
 			require.NoError(t, err)
 			assert.Equal(t, []string{polID}, ids)
 
-			assertEmpty(t, db.QueryRow(`SELECT count(*) FROM iam_policy_members WHERE member_id=$1`, member0.ID))
-			assertOne(t, db.QueryRow(`SELECT count(*) FROM iam_policy_members WHERE member_id=$1`, member1.ID))
+			assertEmpty(t, db.QueryRow(`SELECT count(*) FROM iam_policy_members WHERE member_id=member_db_id($1)`, member0.ID))
+			assertOne(t, db.QueryRow(`SELECT count(*) FROM iam_policy_members WHERE member_id=member_db_id($1)`, member1.ID))
```

Comment:
I wonder if we could use some more helpers, like
```go
func count(from, query string, args ...interface{}) *db.Row {
  return db.QueryRow(fmt.Sprintf("SELECT count(*) FROM %s WHERE %s", from, query), args...)
}
```
so we could turn these into
```go
assertOne(t, count("iam_policy_members", "member_id=member_db_id($1)", member1.ID)
```
🤔 (Generally, sprintf-ing SQL queries is a bad idea, but for tests, fine 🤷‍♂) -- ⚠️ I haven't tested this, the var args handling could be wrong, see https://golang.org/ref/spec#Passing_arguments_to_..._parameters for all the details.

Reply:
yeah fixing up the assertions is also on my list https://chefio.slack.com/archives/C61F9HHKK/p1562111196290600

I'll just put all this in the PR description instead of just hoping ppl stumble on it in slack lol